Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:24.04
+
+# Set default shell during Docker image build to bash
+SHELL ["/bin/bash", "-c"]
+
+# Set non-interactive frontend for apt-get to skip any user confirmations
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN mkdir -p /workspace/
```

Comment:
Maybe using `WORKDIR` is better. 
https://docs.docker.com/reference/dockerfile/